tests/serverpod_test_server/test_e2e/signout/auth_status_signout_test.dart

```diff
@@ -0,0 +1,155 @@
+import 'package:serverpod_test_client/serverpod_test_client.dart';
+import 'package:serverpod_test_server/test_util/test_key_manager.dart';
+import 'package:serverpod_test_server/test_util/config.dart';
+import 'package:test/expect.dart';
+import 'package:test/scaffolding.dart';
+
+void main() {
```

Comment:
Question: What is the difference between the tests in this file and the tests in `auth_signout_test.dart`?
Should the other file maybe be removed?

Reply:
Yes, you're right